src/App.js

```diff
@@ -1,27 +1,13 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends React.PureComponent {
+  render() {
+    return (
+      <div>
+        <Calculator />
+      </div>
+    );
+  }
```

Comment:
- According to the project requirements, please ensure that this component is a functional component, please fix this 🙏